sql/internet_outages/global_outages_v1/query.sql

```diff
@@ -0,0 +1,525 @@
+-- Note: udf.udf_json_extract_int_map map doesn't work in this case as it expects an INT -> INT
+-- map, while we have a STRING->int map
+CREATE TEMP FUNCTION udf_json_extract_string_to_int_map(input STRING) AS (
+  ARRAY(
+    SELECT
+      STRUCT(
+        CAST(SPLIT(entry, ':')[OFFSET(0)] AS STRING) AS key,
+        CAST(SPLIT(entry, ':')[OFFSET(1)] AS INT64) AS value
+      )
+    FROM
+      UNNEST(SPLIT(REPLACE(TRIM(input, '{}'), '"', ''), ',')) AS entry
+    WHERE
+      LENGTH(entry) > 0
+  )
+);
+
+-- TODO: add docs
+CREATE TEMP FUNCTION sum_values(x ARRAY<STRUCT<key INT64, value INT64>>) AS (
+  (
+    WITH a AS (
+      SELECT
+        IF(array_length(x) > 0, 1, 0) AS isPres1
+    ),
+    b AS (
+      SELECT
+        sum(value) AS t
+      FROM
+        UNNEST(x)
+      WHERE
+        key > 0
+    )
+    SELECT
+      coalesce(isPres1 * t, 0)
+    FROM
+      a,
+      b
+  )
+);
+
+-- TODO: docs
```

Comment:
@saptarshiguha could you kindly give me a couple of sentences to add as docs, here?

Reply:
This sums the values for the histogram for buckets >0. If the histogram is missing (isPres1 = FALSE), we return 0